packages/desktop-client/package.json

```diff
@@ -40,7 +43,6 @@
     "memoize-one": "^6.0.0",
     "pikaday": "1.8.0",
     "react": "18.2.0",
-    "react-app-rewired": "^2.2.1",
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`react-scripts` can also be removed IMO

Reply:
Craco still uses react-scripts under the hood I believe: https://craco.js.org/docs/getting-started/